service_manifest.yml

```diff
@@ -1,17 +1,8 @@
-name: IntezerDynamic
+name: Intezer
 version: $SERVICE_TAG
 description: Fetch the Intezer results of the submitted file's sha256
-
-# TODO: From the user guide
-# The following file formats are currently supported:
-# • Windows Executable Files (PE) – exe, .dll, .sys – native x86, native x64 and .NET.
-# • Linux Executable Files (ELF) – native x86, native x64, ARM32, ARM64
-# • Compressed files that contain one file - Zip, RAR, TAR, 7-Zip
-# • Android applications (APK)• Installers - msi, trusted installer, Inno setup...
-# • Microsoft Office - doc, xls, ppt, etc.
-# • PDF
-# • Scripts - powershell, vbs, js
-accepts: .*
+# These are the file type categories that we have found Intezer Analyze hits on.
+accepts: android/.*|executable/.*|document/installer/windows
```

Comment:
You may have a surprise with executable/web/wasm. Are you interested in those too?
I'm assuming you didn't want the rest that was listed in the previous comment.